spec/streamflow/pm.md

```diff
@@ -0,0 +1,408 @@
+
+The allocation value is also the maximum amount that a registered orchestrator will be willing to "float" for a broadcaster. When an orchestrator receives a winning ticket it treats the ticket face value as float since the broadcaster may or may not have sufficient deposit funds to cover the ticket face value at the time of redemption. An orchestrator can safely receive winning tickets and add to its float for a broadcaster up to the allocation value committed to the orchestrator from the broadcaster's reserve. Whenever an orchestrator successfully redeems a winning ticket which draws from a broadcaster's deposit, it can subtract the ticket face value from its float for a broadcaster.
+
+| Field                       | Type                         | Description                                                                               |
+| --------------------------- | ---------------------------- | ----------------------------------------------------------------------------------------- |
+| **fundsAtCreation**         | uint256                      | Funds that the reserve was created with                                                   |
+| **fundsRemaining**          | uint256                      | Funds remaining in the reserve taking into account funds claimed if the reserve is frozen |
+| **freezeRound**             | uint256                      | The round that the reserve is frozen because the sender overspent from its deposit        |
+| **recipientsInFreezeRound** | uint256                      | Number of recipients (i.e. registered orchestrators) during `freezeRound`                 |
+| **claimed**                 | mapping (address => uint256) | Amount claimed by an ETH address (only eligible recipients during `freezeRound`)          |
+
+A broadcaster only has a single reserve at any given point in time.
+
+A broadcaster's reserve can be thought of as a fixed amount of funds that is committed to the set of orchestrators which is updated each round. So, at the beginning of each round, the reserve is split into equal allocations based on the current orchestrator set. Thus, without adding additional funds to a reserve, it will be split into smaller allocations as the orchestrator set approaches its maximum size and it will be split into larger allocations as the orchestrator set shrinks in size. Since each allocation represents an orchestrator's max float for the broadcaster, as the allocation size increase, an orchestrator will be able to safely receive winning tickets with higher face values or more winning tickets with lower face values prior to having to redeem them.
```

Comment:
Can we also explain the incentive for `B` to send the most recent `currentRound` to `O`?

Reply:
I added a few notes on this [here](https://github.com/livepeer/wiki/pull/36/files#diff-37a2c87fbfadd3ec58f7d2989b660216R52). Let me know if you think this is clear enough